src/hocon_postprocess.erl

```diff
@@ -187,6 +187,8 @@ do_duration(Str, Sum) ->
             end
     end.
 
+calc_duration(Val, "f")  -> Val * ?FORTNIGHT;
```

Comment:
this is just to pass the tests in cuttlefish
https://github.com/emqx/cuttlefish/blob/83887e3ef71d65ec6a066726c97b2bbce87f7800/src/cuttlefish_duration.erl#L138